code/modules/mob/living/carbon/human/species/drask.dm

```diff
@@ -56,7 +56,7 @@
 	has_organ = list(
 		"heart" =      				/obj/item/organ/internal/heart/drask,
 		"lungs" =     				/obj/item/organ/internal/lungs/drask,
-		"metabolic strainer" =      /obj/item/organ/internal/liver/drask,
+		"liver" =      /obj/item/organ/internal/liver/drask,
 		"eyes" =     				/obj/item/organ/internal/eyes/drask, //5 darksight.
 		"brain" =  					/obj/item/organ/internal/brain/drask
 		)
```

Comment:
Same issue here too, although not quite as bad:
![image](https://user-images.githubusercontent.com/57483089/130307834-345add32-424d-4f61-a830-321e9bc14b1a.png)